src/main/java/seedu/duke/data/TransactionList.java

```diff
@@ -235,7 +235,7 @@ public String findTransactions(String keywords) {
         // Loops each transaction from the transactions list
         for (Transaction transaction : transactions) {
             // Includes only transactions that contain the keywords used in the search expression
-            if (transaction.toString().contains(keywords)) {
+            if (transaction.getDescription().toLowerCase().contains(keywords.toLowerCase())) {
```

Comment:
Cool! One line changed and two issues solved! Don't forget to update the help command, ug and potentially dg on the new change.

Reply:
Thank you and noted!!